cli/command/swarm/ca.go

```diff
@@ -0,0 +1,133 @@
+package swarm
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"io/ioutil"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/command/swarm/progress"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/pkg/jsonmessage"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type caOptions struct {
+	swarmOptions
+	rootCACert PEMFile
+	rootCAKey  PEMFile
+	rotate     bool
+	detach     bool
+	quiet      bool
+}
+
+func newRotateCACommand(dockerCli command.Cli) *cobra.Command {
+	opts := caOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "ca [OPTIONS]",
+		Short: "Manage root CA",
+		Args:  cli.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runRotateCA(dockerCli, cmd.Flags(), opts)
+		},
```

Comment:
Can you add

```go
Tags: map[string]string{"version": "1.30"},
```

like we have on [cli/command/system/prune.go#L31](https://github.com/docker/cli/blob/d7f6563efc059dcc59b21f30f68e9e8282481af9/cli/command/system/prune.go#L31), so that the command is hidden when talking to an older daemon, and produces an error that it requires a newer daemon